addons/mail/static/src/components/discuss/tests/discuss_tests.js

```diff
@@ -1252,6 +1252,62 @@ QUnit.test('load single message from channel initially', async function (assert)
     );
 });
 
+QUnit.test('load a message from channel with an number as active_id', async function (assert) {
+    assert.expect(4);
+
+    // channel expected to be rendered, with a random unique id that will be referenced in the test
+    this.data['mail.channel'].records.push({ id: 20 });
+    this.data['mail.message'].records.push({
+        channel_ids: [20],
+        date: "2019-04-20 10:00:00",
+        id: 100,
+        model: 'mail.channel',
+        res_id: 20,
+    });
+
+    await this.start({
+        discuss: {
+            params: {
+                default_active_id: 20,
+            },
+        }
+    });
+
+    assert.containsOnce(
+        document.body,
+        `
+            .o_DiscussSidebarItem.o-active[data-thread-local-id="${
+                this.env.models['mail.thread'].find(thread => thread.id === 20).localId
+            }"]
+        `,
+        "should have a load the correct active thread inside the sidebar"
+    );
+    assert.containsOnce(
+        document.body,
+        `
+            .o_Discuss_thread[data-thread-local-id="${
+                this.env.models['mail.thread'].find(thread => thread.id === 20).localId
+            }"]
+        `,
+        "should have a load the correct discuss thread"
+    );
```

Comment:
Just keep this assertion. Test will be straight to the point, very simple and easy to maintain. Right now too cumbersome.